.github/templates/linux_ci_workflow.yml.in

```diff
@@ -0,0 +1,140 @@
+# WARNING THIS IS AUTOGENERATED DO NOT UPDATE MANUALLY PLS
+# Template is at:    .github/templates/linux_ci_workflow.yml
+# Generation script: .github/scripts/generate_linux_ci_workflows.py
+name: Linux CI (!{{ build_environment }})
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+      - release/*
+  workflow_dispatch:
```

Comment:
If you're going to have a separate workflow yaml per linux/windows/etc, you're almost certainly going to want to templatize the `on` block

Reply:
in what way do you mean?